tox.ini

```diff
@@ -267,6 +267,7 @@ deps =
     PyYAML==5.4.1
 commands = python {toxinidir}/scripts/check_doc_links.py
            python {toxinidir}/scripts/check_doc_ipfs_hashes.py
+           python {toxinidir}/scripts/check_ipfs_hashes_pushed.py
```

Comment:
I dont see where you pass a git commit or anything?
Does this test run on every push to main?

Reply:
This runs with every push, yes. The script reads the latest tag from git, gets the corresponding hashes.csv and uses that.

Reply:
This is not gonna work very well. 
We cant push every set of packages. Think of how much storage that would take up. Wed bankrupt the company :p

Reply:
Doesn't make sense to run it then. It should be run before we deploy new docs. @8ball030 could run it in infra when deploying new docs?

Reply:
> This is not gonna work very well. We cant push every set of packages. Think of how much storage that would take up. Wed bankrupt the company :p

This runs against the latest TAG, not hash. We can move it to infra if you prefer, if it can be run automated. My initial thought was that this would be seen by more eyes once it starts failing in PRs so it would be easier to catch. But if we have a way to integrate it into our deployment flow that's fine by me.